src/main/java/hu/oe/nik/szfmv/visualization/Dashboard.java

```diff
@@ -12,6 +12,12 @@
     private final int height = 700;
     private final int backgroundColor = 0x888888;
 
+    private PedalBar bPB = new PedalBar();
+    private JLabel brakePedalLabel = bPB.getPedalProgressBarLabel(40, 290, "Brake pedal");
```

Comment:
A pozíciókat én kiszervezném egy final változóba (ez vonatkozik a következő három mezőre is)

Reply:
👍